### DISCUSSION TITLE
How to set SSL certificates on `vercel dev` for local deployment ?

### DISCUSSION THREAD

**Discussion by qpre** [vercel/vercel#5114] [Help]
Hello !

I'm working on a use case where I'd need Vercel functions to be addressable over https on localhost (a dark story of cookies in between localhost:1234 and localhost:3000) to allow SameSite policy.

How would you achieve that ?

**Comment by paradise-point** [comment]
I got https SSL working on localhost with `vercel dev` using `mkcert` and `local-ssl-proxy` via the following executed by `npm run dev`

```
concurrently 'cp -v _dev.env .env && svelte-kit dev --host --port $PORT' 
'local-ssl-proxy --key localhost-key.pem --cert localhost.pem --source 3001 --target 3000'
```
